galaxy_api/api/migrations/0006_auto_20200119_1627.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 2.2.8 on 2020-01-19 16:27
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('galaxy_api', '0005_collectionimport'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='namespace',
+            name='groups',
+        ),
+        migrations.AddField(
+            model_name='namespace',
+            name='groups',
+            field=models.CharField(default='', max_length=256),
+        ),
```

Comment:
This migration is destructive. It creates empty field for all existing namespaces.